{{ cookiecutter.repo_name }}/{{ cookiecutter.repo_name }}/logging.yml

```diff
@@ -0,0 +1,24 @@
+version: 1
+disable_existing_loggers: true
+# Configuring the default (root) logger is highly recommended
+root:
+  level: INFO
+  handlers: [console]
+loggers:
+  # Logging from application
+  {{cookiecutter.repo_name}}:
+    level: INFO
+    handlers: [console]
+    qualname: {{cookiecutter.repo_name}}
+    propagate: false
+handlers:
```

Comment:
This is so cool that I want more. :D
Could you please add another handler that writes to file? It shouldn't be added to any logger by default, but it would be super easy to add it. This is something I always wanted to have but never had time to do it: to have log messages displayed on console AND saved to file. Please make my dream come true. :pray:

Reply:
Glad you like it :)
I've been experimenting shortly with this even had it in there. But I decided to not use it as you are left with two options:

- map some shared logdir into container -> as it is the same for all containers they all write to the same file or even overwrite it?
- don't map logdir and retrieve logs from container somehow -> Too complicated... I prefer instead to use docker logs.

So I decided that what [docker logs](https://stackoverflow.com/questions/33017329/where-is-a-log-file-with-logs-from-a-container) gives us is enough to handle log files.

During this experimentation I have found some issues. Most of them I could resolve I'm still left with the issue that the command `docker-compose run controller luigi ..` currently logs everything double, but container logs show only a single log line.